addon/webextension/background/selectorLoader.js

```diff
@@ -64,6 +64,11 @@ this.selectorLoader = (function() {
   let loadingTabs = new Set();
 
   exports.loadModules = function(tabId, hasSeenOnboarding) {
+    // Sometimes loadModules is called when the tab is already loading.
+    // Adding a check here seems to help avoid catastrophe.
+    if (loadingTabs.has(tabId)) {
```

Comment:
Won't this get in the way of loading modules on the same tab, when it's a new page? E.g., just reload the page and try to re-invoke screenshots, and then this check will stop it from loading...?